src/grammar.ne

```diff
@@ -88,7 +88,7 @@ resistor ->
 
 rSpecs -> (_ rSpec _):* | __
 
-rSpec -> tolerance | power_rating
+rSpec -> tolerance | J U M P E R {% () => ({ tolerance: 0 }) %} | power_rating
```

Comment:
Can you explain your reasoning here? Do we need this jumper keyword?

Reply:
Could you explain your use case a bit more? To me this change lets people write "1k jumper" which would be the same as "1k 0%" but that doesn't make much sense. "0 ohm" would still be a jumper and a jumper doesn't have a tolerance. Why not just ignore the word "jumper"?

Reply:
We wanted to make the parts as comprehensive as possible, and therefore we store a tolerance value also for jumpers. In my opinion, 0 is the right value for this. You are totally correct, this version would allow people to state 0 tolerance in a different/semantically incorrect way. Questions here is to what extent the grammar should cope with this semantic aspect anyway. I mean, 0% tolerance, really? If you think the jumper thing should not be included, I can revert this commit.

Reply:
Yeah, please take out the jumper change. It's just not making sense to me. I don't think 0 tolerance is the right way to describe it and I think your use will be fine without it when it's ignored?  

I am curious what you are working on. Can you give me some info?

Reply:
Ah, just noticed you are working for @AislerHQ

Reply:
Yep. Here you go.